siliconcompiler/utils/logging.py

```
@@ -29,12 +29,12 @@ def __init__(self):
 
 
 class SCDebugInRunLoggerFormatter(logging.Formatter):
-    def __init__(self, chip, jobname, step, index):
+    def __init__(self, project, jobname, step, index):
```

Ruff (0.13.1)

Unused method argument: `jobname`

(ARG002)